R/Model-validity.R

```diff
@@ -76,6 +76,44 @@ v_model_logistic_kadane <- function(object) {
   v$result()
 }
 
+#' @describeIn v_model_objects validates that the logistic Kadane model
+#'   parameters with a beta and gamma prior are valid.
+v_model_logistic_kadane_beta_gamma <- function(object) {
+  v <- Validate()
+  v$check(
+    is.probability(object@theta, bounds = FALSE),
+    "theta must be a probability scalar > 0 and < 1"
+  )
+  is_xmin_number <- test_number(object@xmin)
+  v$check(is_xmin_number, "xmin must be scalar")
+
+  is_xmax_number <- test_number(object@xmax)
+  v$check(is_xmax_number, "xmax must be scalar")
+
+  if (is_xmin_number && is_xmax_number) {
+    v$check(object@xmin < object@xmax, "xmin must be strictly smaller than xmax")
+  }
```

Comment:
no need to check these and theta as it is done in LogisticKadane already

Reply:
ok I also then removed any unit tests related to these slots from the test-Model-Validity using the v_model_logisitc_kadane_beta_gamma()

In addition I have repeated 3 test for v_model_logisitc_kadane() using as input the h_get_logistic_kadane_beta_gam() instead of h_get_logistic_kadane()